issue found on: server/server.go:55:2: importShadow

an issue has been detected in source code: ./server/server.go:55:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-results-aggregator-mock/content' package 'content'

--- a/server/server.go
+++ b/server/server.go
@@ -52,12 +52,12 @@ type HTTPServer struct {
 func New(config Configuration,
 	storageInstance storage.Storage,
 	groups map[string]groups.Group,
-	content []content.RuleContent) *HTTPServer {
+	ruleContents []content.RuleContent) *HTTPServer {
 	return &HTTPServer{
 		Config:  config,
 		Storage: storageInstance,
 		Groups:  groups,
-		Content: content,
+		Content: ruleContents,
 	}
 }
 

